tests/tests_api/test_age_range.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from rest_framework import status
+
+from api.models.age_ranges import AgeRange, AgeRangeType
+
+
+@pytest.mark.parametrize(
+    "age_range_type",
+    [AgeRangeType.STUDENT, AgeRangeType.TEACHER, AgeRangeType.MATCHING, None],
+)
+def test_age_range_list(admin_client, age_range_type):
+    queryset = AgeRange.objects.order_by("id")
+    if age_range_type:
+        queryset = queryset.filter(type=age_range_type)
+        query_string = f"?type={age_range_type}"
+    else:
+        query_string = ""
+    response = admin_client.get(f"/api/age_ranges/{query_string}")
```

Comment:
is trailing slash important? Some of the tests have it after `query_string`, some don't. I guess we should be consistent.

Reply:
Yes, It's important. For url without query string there are should be trailing slash, and with query string - no. Otherwise, tests won't pass :)

Reply:
OK yes, right